### DISCUSSION TITLE
Open terminal from login screen

### DISCUSSION THREAD

**Discussion by etkaar** [elementary/greeter#496] [Q&A] [answered]
Is there a way to open a terminal from the login screen? This could be useful for cases like that:

![photo_2020-12-28_06-54-58](https://user-images.githubusercontent.com/40885610/103195164-080fe400-48e2-11eb-92a5-08a11ece3e40.jpg)

**Comment by danirabbit** [comment, marked as answer]
There isn't a way to open the Terminal app on the greeter, but **you can drop to a fullscreen TTY with the keyboard shortcut `Ctrl` + `Alt` + `F6`**. You can switch between various available TTYs by using that keyboard shortcut with `F1`-`F6`. `Ctrl` + `Alt` + `F7` will bring you back to the greeter

**Comment by etkaar** [comment]
@danrabbit Thanks a lot for the fast answer!

Indeed, `Ctrl+Alt+F1-F6` can do the trick - however, this is not as simple as it sounds, because unfortunately in many cases this will not work due to driver problems (it will end in a simple [blackscreen](https://askubuntu.com/questions/162535/why-does-switching-to-the-tty-give-me-a-blank-screen)). There may be ways to fix that by editing or updating grub, but I personally had no chance. I needed to do a more or less complex and time intense [rescue](https://blog.sleeplessbeastie.eu/2015/11/16/how-to-mount-encrypted-lvm-logical-volume/) to fix the simple problem behind. In my case, I accidentially removed required packages, so `rm ~/.Xauthority && apt install pantheon-xsession-settings` did the trick.

That in mind, I feel that it would be a very helpful feature to open a terminal in the login/lock screen for such cases where you are completely locked out from the system. If not by an clickable small icon, then using a shortcut.

So, shouldn't we move that to issues/ideas rather than keeping it in Q&A?

**Reply by danirabbit** [reply]
If you'd like to open a feature request, please feel free to do so in the "Issues" tab. Please make sure to use the template when filing :+1: 

I'm going to hide this comment as "Off topic" since it is not a proposed answer to the original question.
